silo-to-silo/integration-tests/aurora-sdk-integration-tests/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "aurora-sdk-integration-tests"
```

Comment:
Let's contribute to the sdk repo instead of copy-pasting it

Reply:
fixed: https://github.com/aurora-is-near/rainbow-token-connector/pull/226/commits/585b9859142a93201ec16a5f49c75a80f8159be6